assets/src/js/admin/stripe-admin.js

```diff
@@ -15,13 +18,41 @@ window.addEventListener( 'DOMContentLoaded', function() {
 	const stripeDisconnect = document.querySelector( '.give-stripe-disconnect' );
 	const checkoutTypes = document.querySelectorAll( 'input[name="stripe_checkout_type"]' );
 	const legacyCheckoutFields = Array.from( document.querySelectorAll( '.stripe-checkout-field' ) );
+	const stripeConnectedElement = document.getElementById( 'give-stripe-connect' );
 
 	giveStripeJsonFormattedTextarea( stripeStylesBase );
 	giveStripeJsonFormattedTextarea( stripeStylesEmpty );
 	giveStripeJsonFormattedTextarea( stripeStylesInvalid );
 	giveStripeJsonFormattedTextarea( stripeStylesComplete );
 	giveStripeJsonFormattedTextarea( stripeCustomFonts );
 
+	if ( null !== stripeConnectedElement ) {
+		const stripeStatus = stripeConnectedElement.getAttribute( 'data-status' );
+		const redirectUrl = stripeConnectedElement.getAttribute( 'data-redirect-url' );
+		const canDisplay = stripeConnectedElement.getAttribute( 'data-display' );
+		const modalTitle = stripeConnectedElement.getAttribute( 'data-title' );
+		const modalFirstDetail = stripeConnectedElement.getAttribute( 'data-first-detail' );
+		const modalSecondDetail = stripeConnectedElement.getAttribute( 'data-second-detail' );
+
+		if ( 'connected' === stripeStatus && '0' === canDisplay ) {
+			new GiveConfirmModal(
+				{
+					modalWrapper: 'give-stripe-connected-modal give-modal--success',
+					modalContent: {
+						title: modalTitle,
+						desc: '<span>' + modalFirstDetail + '</span>' + '<span class="give-field-description">' + modalSecondDetail + '</span>',
```

Comment:
@mehul0810 use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
@ravinderk  Used Template Literals. Thanks for the suggestion.